Traceback (most recent call last):
  File "/Users/apple/Desktop/DataEngineeringBootCamp/GithubProfile/jackie-yt-wang/scripts/update_weather.py", line 39, in <module>
    with open(readme_path, "r+") as file:
         ^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/apple/Desktop/DataEngineeringBootCamp/GithubProfile/jackie-yt-wang/.github/workflows/README.md'
dule>
    with open(readme_path, "r+") as file:
         ^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/apple/Desktop/DataEngineeringBootCamp/GithubProfile/jackie-yt-wang/.github/workflows/README.md'
PYTHON RUNNING FAILED
[ERROR:] RETURN CODE:  1
[ERROR:] REFER TO THE LOG FOR THE REASON FOR THE FAILURE.
